taqueria-plugin-taquito/README.md

```diff
@@ -13,35 +13,32 @@ taq install @taqueria/plugin-taquito
 
 The target networks, sandboxes, and environments are configured in the Taqueria project's `config.json` file. For additional information on configuring network, documentation can be found [here](/docs/config/networks/)
 
-## Usage
-
-###  The `taq originate` task
-
-The Taquito plugin exposes an `taq originate` task in Taqueria which will originate the specified Michelson contract to a Taqueria environment
+##  The `taq originate` task
 
 Basic usage is:
 
 ```shell
-taq originate
+taq originate <filename>
 ```
 
-This will originate all '.tz' files in the `/artifacts` directory to the default environment (the sandbox named `local`)
+### Basic description
 
-To target a different environment, use the `--environment` flag with the named Taqueria environment you want to target :
+The Taquito plugin exposes an `taq originate` task in Taqueria which will originate the specified Michelson contract to a Taqueria environment.
 
-```shell
-taq originate -e jakartanetEnv
-```
+This will originate a Michelson `.tz` file in the `/artifacts` directory to the default environment (the sandbox named `local`).
 
-## Plugin Architecture
+By default, the storage value used for origination will live in a file named `CONTRACT.default.storage.tz`, in `/artifacts`, where `CONTRACT` is the name of the contract to originate. If it can't find it, it'll not originate the contract. Alternatively, you can explicitly specify a storage file with the `--storage` option described below.
```

Comment:
`CONTRACT.default.storage.tz`
should it be `CONTRACT.default_storage.tz`?

Reply:
Thanks @AlirezaHaghshenas ! yep yep